.github/README_fr.md

```diff
@@ -0,0 +1,45 @@
+# HeliPort
+
+![GiHub Actions](https://github.com/zxystd/HeliPort/workflows/Build%20and%20Test/badge.svg?branch=master)
+
+Client Wifi Intel pour [itlwm](https://github.com/zxystd/itlwm)
+
+## Discussion
+
+Si vous avez des questions ou que vous avez des retours à faire n'hesitez pas à nous rejoindre : [![Rejoignez le chat at https://gitter.im/OpenIntelWireless/itlwm](https://badges.gitter.im/OpenIntelWireless/itlwm.svg)](https://gitter.im/OpenIntelWireless/itlwm?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+
+## Statut
+
+Le projet en est encore à ses débuts, seul l'interface fonctionne pour le moment.
+
+Visitez [Projects](https://github.com/zxystd/HeliPort/projects) pour plus d'informations
+
+## Problèmes
+
+Les problèmes remontés pour ce projet doivent être liés à des bugs ainsi qu'aux demandes d'évolution, prenez le temps de remplir tous les termes dans le template de report de bug. 
+
+Les problèmes suivants seront considérés comme invalides et fermés immédiatement:
+
+- Demande d'aide personnel
+- Problèmes dupliqués
+- Demandes de maj urgentes
+- Spam
+- Contenus non-anglais/non-chinois
+- Questions facilement trouvable via Google
+  > Comment build etc.
+- Hors sujets
+  > Incluant la mention et la distribution des projets.
+
+## Contribution
+
+Nous avons désespérément besoin de contributeurs pour nous aider à améliorer le projet, tout aide sera grandement appréciée.  
+- Les interfaces utilisateur et la gestion de mots de passe sont implémentés en `Swift 5`
```

Comment:
Missing newline between those two lines

[MD032/blanks-around-lists: Lists should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.20.3/doc/Rules.md#md032)